content/applications/sales/crm/pipeline/multi_sales_team.rst

```diff
@@ -1,47 +1,66 @@
-===========================
-Manage multiple sales teams
-===========================
+====================
+Multiple sales teams
+====================
 
-In Odoo, you can manage several sales teams, departments or channels
-with specific sales processes. To do so, we use the concept of *Sales
-Channel*.
+In Odoo, you can manage several sales teams, departments, or channels, each with their own unique
+sales processes, using *Sales Teams*.
 
-Create a new sales channel
-==========================
 
-To create a new *Sales Channel*, go to :menuselection:`Configuration --> Sales Channels`.
+Create a new Sales Team
+=======================
 
-There you can set an email alias to it. Every message sent to that email
-address will create a lead/opportunity.
+To create a new *Sales Team*, go to :menuselection:`CRM --> Configuration --> Sales Teams` then
```

Comment:
I don't think this instance of "Sales Team" needs to be italicized. Let's italicize it once at the very beginning for emphasis/understanding, then leave it unitalicized but capitalized in the rest of the doc.

`To create a new Sales Team, go to :menuselection:CRM --> Configuration --> Sales Teams then`

Please note that I removed the backticks around the menuselection items in my corrected quote.